src/ImageSequencer.js

```diff
@@ -292,6 +299,30 @@ ImageSequencer = function ImageSequencer(options) {
     });
   }
 
+  function loadNewModule(name,options){
+
+    if(!options) {
+      return this;
+    }
+    else if(Array.isArray(options)) {
+      // contains the array of module and info
+      this.module[name] = options;
+
+    } else  if(options.func && options.info) {
+      // passed in options object
+      this.modules[name] = [
+        options.func, options.info
+      ]
+    } else if(options.path&&!this.inBrowser) {
+      // load from path(only in node)
+      const module = [
+        require(`${options.path}/Module.js`), require(`${options.path}/info.json`)
```

Comment:
Let's put each `require()` on a separate line for readability. :+1:

Reply:
done